lms/djangoapps/learner_dashboard/urls.py

```diff
@@ -6,5 +6,6 @@
 from . import views
 
 urlpatterns = [
+    url(r'^programs/(?P<programId>[0-9]+)/$', views.view_program_details, name='program_details_view'),
```

Comment:
@AlasdairSwan I notice one requirement from the engineering side not really captured in the Jira ticket: we want the programId be the UUID of the program instead of the Database Identifier (what you have now). So, the URL pattern here needs to be updated to accept a UUID